TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -8,13 +8,20 @@
 
 import UIKit
 
+protocol LargeDetailTableViewDelegate: class {
+    func collapseCells(indexPath: IndexPath)
+    func expandCells(indexPath: IndexPath)
+}
+
 class LargeDetailTableViewCell: UITableViewCell {
 
     var iconView: DetailIconView!
     var titleLabel: UILabel!
     var detailLabel: UILabel!
     var busIconView: BusIcon!
-    var chevron: UIImageView!
+    var chevron: UIButton!
+    weak var delegate: LargeDetailTableViewDelegate?
+    var indexPath: IndexPath!
```

Comment:
following up on this, you could then get rid of this variable bc you can get the indexPath from the tableview (and this kind of breaks MVC)